PluginFramework/Formatter/BaseFormatter.cs

```diff
@@ -46,14 +45,14 @@ public bool ConvertToFormat(string filePath, string outputName, ModesEnum mode)
         /// <inheritdoc/>
         public void ChangeSettings(PluginSettings settings)
         {
-            this.Settings = settings;
+            Settings = settings;
         }
 
         /// <inheritdoc/>
-        public virtual UserControl GetSettingsPage()
-        {
-            return null;
-        }
+        //public virtual UserControl GetSettingsPage()
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)